adapters/ix/ix.go

```diff
@@ -417,6 +420,28 @@ func (a *IxAdapter) MakeBids(internalRequest *openrtb2.BidRequest, externalReque
 				}
 			}
 
+			var bidNative1_1 *Native11Wrapper
```

Comment:
Could you please change the names of the variables `bidNative1_1` and `bidNative1_2` so they follow the Go naming convention (no underscores)? Maybe something like `bidNative1v1` and `bidNative1v2` would work.

Reply:
Updated!